frontend/src/metabase/visualizations/visualizations/CartesianChart/constants.ts

```diff
@@ -0,0 +1,18 @@
+import { t } from "ttag";
+
+import type { CartesianChartDateTimeAbsoluteUnit } from "metabase/visualizations/echarts/cartesian/model/types";
+
+export const DATETIME_ABSOLUTE_UNIT_COMPARISON: Record<
+  CartesianChartDateTimeAbsoluteUnit,
+  string
+> = {
+  year: t`Compared to previous year`,
+  quarter: t`Compared to previous quarter`,
```

Comment:
On quarters it showed months:
<img width="1356" alt="Screenshot 2024-05-03 at 4 31 32 PM" src="https://github.com/metabase/metabase/assets/14301985/97eca402-577a-4843-a585-f0aafaedac61">